auxx/src/Plugin.hs

```diff
@@ -149,35 +141,15 @@ withValueText cont = \case
     Lang.ValueList vs -> for_ vs $
         withValueText (cont . mappend "  ")
 
-
-
-----------------------------------------------------------------------------
--- Something hacky
```

Comment:
Indeed. `OutSpecs` on workers now clearly make no sense, since `OutSpecs` are only relevant inside the full diffusion layer.